tests/unit/AssociationType.spec.ts

```diff
@@ -0,0 +1,75 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import VueRouter from 'vue-router'
+import flushPromises from 'flush-promises'
+import sinon from 'sinon'
+import { getVuexStore } from '@/store/'
+import { shallowMount, mount, createLocalVue } from '@vue/test-utils'
+import mockRouter from './MockRouter'
+import { AssociationType } from '@/components/common'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const store = getVuexStore()
+
+// Prevent the warning "[Vuetify] Unable to locate target [data-app]"
+document.body.setAttribute('data-app', 'true')
+
+describe('AssociationType component', () => {
+  let wrapper: any
+  const { assign } = window.location
```

Comment:
All this `window.location` stuff is to test routing. I think this component is not involved in routing so you can clean this all up.

Reply:
Ditto with local Vue object below.

(KISS)

Reply:
This can be cleaned up further since this suite doesn't test routing (ie, remove localVue). Can be done later. OK for now.